pod/perldelta.pod

```diff
@@ -338,51 +935,500 @@ XXX
 
 =head1 Internal Changes
 
-XXX Changes which affect the interface available to C<XS> code go here. Other
-significant internal changes for future core maintainers should be noted as
-well.
+=over 4
 
-[ List each change as an =item entry ]
+=item *
 
-=over 4
+The L<perlapi/C<sv_strftime_ints>> function is introduced.  This is an
+enhanced version of L<perlapi/C<my_strftime>>, which is retained for
+backwards compatibility.  Both are to call L<strftime(3)> when you have
+the year, month, hour, etc.  The new function handles UTF8ness for you,
+and allows you to specify if you want the possibility of daylight
+savings time to be considered.  C<my_strftime> never considers DST.
 
 =item *
 
-XXX
+The C<bytes_to_utf8>, C<bytes_from_utf8>, and C<bytes_from_utf8_loc>
+functions are no longer experimental.
+
+=item *
+
+Calls to L<call_argv()|perlapi/call_argv> with the C<G_DISCARD> flag
+set also ensure the SV parameters constructed from the C<argv>
+parameter are released before C<call_argv()> returns. Previously they
+were released on the next L<FREETMPS|perlapi/FREETMPS>. [L<GH #22255|https://github.com/Perl/perl5/issues/22255>]
+
+=item *
+
+When built with the C<-DDEBUGGING> compile option, perl API functions that
+take pointers to distinct types of SVs (AVs, HVs or CVs) will check the
+C<SvTYPE()> of the passed values to ensure they are valid.  Additionally,
+internal code within core functions that attempts to extract AVs, HVs or CVs
+from reference values passed in will also perform such checks.
+
+While this has been entirely tested by normal Perl CI testing, there may
+still be some corner-cases where these constraints are violated in
+otherwise-valid calls.  These may require further investigation if they are
+found, and specific code to be adjusted to account for it.
+
+=item *
+
+The C<op_dump()> function has been expanded to include additional information
+about the recent C<OP_METHSTART> and C<OP_INITFIELD> ops, as well as for
+C<OP_ARGCHECK> and C<OP_ARGELEM> which had not been done previously.
+
+=item *
+
+C<op_dump()> now also has the facility to print extra debugging information
+about custom operators, if those operators register a helper function via the
+new C<xop_dump> element of the C<XOP> structure. For more information, see the
+relevant additions to L<perlguts|perlguts/"Custom Operators">.
+
+=item *
+
+Enable removing most of mathoms.c and stub functions [L<GH #22691|https://github.com/Perl/perl5/issues/22691>] and [L<GH #22714|https://github.com/Perl/perl5/issues/22714>]
+
+=item *
+
+put apostrophe as package separator behind a default enabled feature,
+which is disabled from feature bundle 5.41.
+
+=item *
+
+pp_reverse: don't COW buffer just to then un-COW it [L<GH #22729|https://github.com/Perl/perl5/issues/22729>]
```

Comment:
This doesn't need to be included. There's no API change, it doesn't change the visible operation of the OP and the performance change won't be noticeable to end users.